### ISSUE TITLE
AI and Machine Learning - The use cases section is missing active verbs to better outline what role AI can play

### ISSUE THREAD

**Issue by joshueoconnor** [w3c/ai-accessibility#9]
The section "AI accessibility use cases" is missing active verbs to give a sense of what role AI can play for each of them e.g.

* /Creating/ alternative text for images
* /Improving/ automatic Speech Recognition for captioning
* /Editing complex text to output/ plain language
* /Performing/  automated Language detection
* /Improving/ poor Colour contrast
* /Improving/ Heading structure (and other roles for content)
* /Proving better context/ for Link purpose
* /Developing automated/ Sign language from text and auditory input

This should help give the reader of what the AI can potentially do for these requirements
